documentation/asciidoc/topics/yaml_annotated/cr_backup.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: infinispan.org/v2alpha1
+kind: Backup <1>
+metadata:
+  name: my-backup <2>
+spec:
+  cluster: source-cluster <3>
```

Comment:
It's possible to selectively backup resources as well, maybe we should add `spec.resources` like in the Restore example?

Reply:
Thanks, Ryan. I've added `spec.resources` as a separate step for both backup and restore procedures. Hopefully it looks alright that way.